pkg/beholder/client.go

```diff
@@ -90,16 +90,20 @@ func newGRPCClient(cfg Config, otlploggrpcNew otlploggrpcFactory) (*Client, erro
 			return nil, err
 		}
 	}
-	sharedLogExporter, err := otlploggrpcNew(
+	opts := []otlploggrpc.Option{
 		otlploggrpc.WithTLSCredentials(creds),
 		otlploggrpc.WithEndpoint(cfg.OtelExporterGRPCEndpoint),
-		otlploggrpc.WithRetry(otlploggrpc.RetryConfig{
+	}
+	if cfg.EmitterExporterRetryConfig != nil {
```

Comment:
By default, the retry is enabled in the OTel SDK. This check is needed to avoid disabling default retry unintentionally when EmitterExporterRetryConfig is not set.